core/sql/regress/compGeneral/TEST042

```diff
@@ -229,6 +229,10 @@ values (600,52,43,4.12345678,6.0932);
 --
 insert into p9tab (c5) values (0E0);
 
+-- caching behavior changes with or without authorization initialized,
+-- make sure authorization is initialized when we run this test
+initialize authorization;
```

Comment:
So, do we get an error if authorization is already initialized? Is the error filtered out of the diffs?